x11-terms/alacritty/alacritty-0.14.0.ebuild

```diff
@@ -0,0 +1,413 @@
+	windows_x86_64_msvc@0.48.5
+	windows_x86_64_msvc@0.52.6
+	winit@0.30.4
+	winnow@0.5.40
+	winnow@0.6.18
+	winreg@0.52.0
+	wio@0.2.2
+	x11-clipboard@0.9.3
+	x11-dl@2.21.0
+	x11rb-protocol@0.13.1
+	x11rb@0.13.1
+	xcursor@0.3.5
+	xdg@2.5.2
+	xkbcommon-dl@0.4.2
+	xkeysym@0.2.1
+	xml-rs@0.8.20
+	yeslogic-fontconfig-sys@5.0.0
+	zerocopy-derive@0.7.35
+	zerocopy@0.7.35
+"
+
+MY_PV="${PV//_rc/-rc}"
+
+inherit bash-completion-r1 cargo desktop
+
+DESCRIPTION="GPU-accelerated terminal emulator"
+HOMEPAGE="https://alacritty.org"
+
+if [ ${PV} == "9999" ] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/alacritty/alacritty"
+else
+	SRC_URI="
+		https://github.com/${PN}/${PN}/archive/refs/tags/v${MY_PV}.tar.gz -> ${P}.tar.gz
+		${CARGO_CRATE_URIS}
+	"
+	KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+fi
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+LICENSE="Apache-2.0"
+# Dependent crate licenses
+LICENSE+="
+	Apache-2.0 BSD-2 BSD Boost-1.0 CC0-1.0 ISC MIT MPL-2.0
+	Unicode-DFS-2016
+"
+SLOT="0"
+IUSE="wayland +X"
+
+REQUIRED_USE="|| ( wayland X )"
+
+COMMON_DEPEND="
+	media-libs/fontconfig:=
+	media-libs/freetype:2
+	x11-libs/libxkbcommon
+	X? ( x11-libs/libxcb:= )
+"
+
+DEPEND="
+	${COMMON_DEPEND}
+"
+
+RDEPEND="
+	${COMMON_DEPEND}
+	media-libs/mesa[X?,wayland?]
+	sys-libs/zlib
+	sys-libs/ncurses:0
+	wayland? ( dev-libs/wayland )
+	X? (
+		x11-libs/libXcursor
+		x11-libs/libXi
+		x11-libs/libXrandr
+	)
+"
+
+BDEPEND="
+	dev-build/cmake
+	>=virtual/rust-1.74.0
+	app-text/scdoc
+"
+
+QA_FLAGS_IGNORED="usr/bin/alacritty"
+
+src_unpack() {
+	if [[ "${PV}" == *9999* ]]; then
+		git-r3_src_unpack
+		cargo_live_src_unpack
+	else
+		cargo_src_unpack
+	fi
+}
+
+src_configure() {
+	local myfeatures=(
+		$(usex X x11 '')
+		$(usev wayland)
+	)
+	cargo_src_configure --no-default-features
+}
+
+src_compile() {
+	scdoc < ./extra/man/alacritty.1.scd > ./alacritty.1 || die
+	scdoc < ./extra/man/alacritty.5.scd > ./alacritty.5 || die
+	scdoc < ./extra/man/alacritty-msg.1.scd > ./alacritty-msg.1 || die
+	scdoc < ./extra/man/alacritty-bindings.5.scd > ./alacritty-bindings.5 || die
+
+	cd alacritty || die
+	cargo_src_compile
+}
+
+src_install() {
+	cargo_src_install --path alacritty
+
+	doman alacritty.1 alacritty.5 alacritty-msg.1 alacritty-bindings.5
+
+	newbashcomp extra/completions/alacritty.bash alacritty
+
+	insinto /usr/share/fish/vendor_completions.d/
+	doins extra/completions/alacritty.fish
+
+	insinto /usr/share/zsh/site-functions
+	doins extra/completions/_alacritty
+
+	domenu extra/linux/Alacritty.desktop
+	newicon extra/logo/compat/alacritty-term.svg Alacritty.svg
+
+	insinto /usr/share/metainfo
+	doins extra/linux/org.alacritty.Alacritty.appdata.xml
+
+	insinto /usr/share/alacritty/scripts
+	doins -r scripts/*
+
+	local DOCS=(
+		CHANGELOG.md README.md
+	)
+	einstalldocs
+}
+
+src_test() {
+	cd alacritty || die
+	cargo_src_test
+}
+
+pkg_postinst() {
+	if [[ -z ${REPLACING_VERSIONS} ]]; then
+		einfo "For information on how to configure alacritty, see the manpage:"
```

Comment:
There was a line about looking into `$HOME/.config` which I've dropped since it's rather confusing, if user action is required app communicates that and tells what to do (it was like that pretty much forever).

Reply:
Could also be a link to our wiki, if a page exists for alacritty.

Reply:
The wiki page is outdated and all the information alacritty carries in its man page.

We also have a https://alacritty.org/config-alacritty.html , but it's just a pre-rendered man page.

Part of the wiki for alacritty on gentoo is about things not related to alacritty configuration.

Though, if you feel like that it'll help the users, go ahead, I'm biased here.